src/_layouts/rss2-feed.njk

```diff
@@ -48,10 +48,10 @@
 		<pubDate>{{ post.date.toUTCString() }}</pubDate>
 		<dc:creator>{{ metadata.author.name }}</dc:creator>
 		<guid isPermaLink="true">{{ absolutePostUrl }}</guid>
-		<description>{{ post.data.excerpt }}</description>
+		<description>{{ post.data.excerpt or "???" }}</description>
```

Comment:
Same here&hellip; I guess some post didn't have an `excerpt` defined in the frontmatter, which caused this to be `undefined` which caused an error; so I added a weak default value.